backend/game/scribble/handler/gamemaster/gamemaster.go

```diff
@@ -181,6 +186,7 @@ func Get(reg registry.Registry) *Handler {
 		playTimeChan:     make(chan PlayTimeChanReceive),
 		selectTopicChan:  make(chan WordSelectReceive),
 		waitForStartChan: make(chan WaitForStartReceive),
+		EndChan:          make(chan bool),
```

Comment:
Same with this one, any particular reason it's public?

Reply:
It was declared exported so I had to match the case, but yes, I'll be changing this when I change the declaration